build.gradle

```diff
@@ -1,3 +1,8 @@
+plugins {
+  id "org.owasp.dependencycheck" version "1.4.5"
+  id "com.github.johnrengelman.shadow" version "1.2.4"
```

Comment:
@shauvik Can you make necessary changes to point the repo back to maven central?